tests/test_generator.py

```diff
@@ -24,3 +24,17 @@ def test_object():
     output = generate_json(schema)
     assert isinstance(output, dict)
     assert isinstance(output.get("a", 0), int)
+
+def test_boolean():
+    """Test generating boolean."""
+    schema = {
+        "type": "object",
+        "properties": {
+            "a": {
+                "type": "boolean"
+            }
+        }
+    }
+    output = generate_json(schema)
+    assert isinstance(output, dict)
```

Comment:
We want unit tests to be as small as possible, so `test_boolean` could be modeled after `test_integer` rather than `test_object`.